src/js/components/AddNewDialog.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div class="addNewDialog">
```

Comment:
nit: I prefer kebab case for html attributes rather than camel-case. We should just adopt a standard, document it, and stick to it.

Reply:
Agreed!  Thanks for catching this.